src/st/no0/e_ctulhu.c

```diff
@@ -114,8 +155,9 @@ void EntityCtulhu(Entity* self) {
             }
             break;
         case 4:
-            AnimateEntity(D_us_801823B8, self);
-            if (self->animFrameIdx == 3 && self->animFrameDuration == 0) {
+            // Fly horizontally for a period
+            AnimateEntity(anim_wing_flap, self);
+            if (!self->animFrameDuration && self->animFrameIdx == 3) {
                 PlaySfxPositional(SFX_WING_FLAP_A);
             }
             colRet = UnkCollisionFunc2(D_us_8018234C);
```

Comment:
Wasn't able to trigger this collision, so unsure what to name it. My guess is maybe it's a roof collision? Didn't seem to happen inside of `no0` anyway